modules/cnf-du-updating-performance-addon-operator.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-performance-addon-operator.adoc_{context}"]
+
+= Updating Performance Addon Operator
+
+After you have completed workload partitioning, you need to ensure that the Performance Addon Operator has been installed and configured on a
```

Comment:
Maybe "After you have configured workload partiioning" and a reference to the section to modules/cnf-du-configuring-workload-partitioning.adoc

Reply:
Changed to "configured". Cannot add internal xrefs at this time.